archive/c/c-plus-plus/README.md

```diff
@@ -14,6 +14,15 @@ Welcome to Sample Programs in C++!
 - Debut: 1985
 - Typing: Static
 
+## Style Guide
+For a long time, the styling that you would use in your code would depend on the company you worked for. Google, Mozilla, Apple, etc have their own style guidelines that their employees must follow for a more consistent codebase. Even individual developers form their own styles.
+
+A set of guidelines has been created by Bjarne Stroustrup, the language creator of C++ and Herb Sutter, a well known C++ expert in the community. The document is currently under continuous development, but it still serves well as an excellent place to read up on the best coding practices for C++ projects.
+
+The guidelines are separated into sections that deals with a language topic such as classes and exceptions.
+
+Link: http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines
```

Comment:
Everything looks good to me, but I was wondering if you could link these more elegantly in a sentence? For instance, maybe you could link this directly in the second paragraph over `set of guidelines`.

Reply:
Done